### PR TITLE
Adding newly generated MGMT examples.

### PR DESCRIPTION
**Pull request by TimSimpsonR**

### CODE REVIEW

File: apidocs/src/resources/samples/db-mgmt-get-host-detail-response.xml [hunk 1]
```diff
@@ -1,11 +1,12 @@
...
+Date: Tue, 24 Jul 2012 19:48:33 GMT
 
-<host name="host" percentUsed="17" totalRAM="6144" usedRAM="1024" xmlns="http://docs.openstack.org/database/api/v1.0">
-	<instances>
-		<instance id="1" name="json_rack_instance" status="active"/>
-		<instance id="2" name="xml_rack_instance" status="active"/>
-	</instances>
+<host name="hostname" percentUsed="51" totalRAM="2004" usedRAM="1024" xmlns="http://docs.openstack.org/database/api/v1.0">
+    <instances>
+        <instance id="6e038c18-902f-4a41-b2a2-5f46fa62fc1b" name="json_rack_instance" server_id="64438fb6-4b98-4bd8-975e-222bc11daf74" status="ACTIVE"/>
```
Comment (ed-) [inline apidocs/src/resources/samples/db-mgmt-get-host-detail-response.xml:8]: These are going to be made obsolete very soon when the tenant_id change gets merged in :C

File: apidocs/src/resources/samples/db-mgmt-get-instance-details-response.json [hunk 2]
```diff
@@ -51,43 +22,30 @@ Date: Wed, 25 Jan 2012 21:58:25 GMT
...
-                "name": "userwith2dbs"
+                "href": "https://ord.databases.api.rackspacecloud.com/instances/6e038c18-902f-4a41-b2a2-5f46fa62fc1b", 
+                "rel": "bookmark"
             }
         ], 
+        "local_id": 0, 
+        "name": "json_rack_instance", 
+        "server_id": "64438fb6-4b98-4bd8-975e-222bc11daf74", 
+        "status": "ACTIVE", 
+        "task_description": "No tasks for the instance.",
```
Comment (ed-) [inline apidocs/src/resources/samples/db-mgmt-get-instance-details-response.json:43]: Hooray

File: apidocs/src/resources/samples/db-mgmt-get-root-details-response.json [hunk 3]
```diff
@@ -1,12 +1,12 @@
...
 
 {
-    "root_enabled_history": {
-        "id": 2, 
-        "root_enabled_at": "2012-01-25 21:58:11", 
-        "root_enabled_by": "examples"
+    "root_history": {
+        "enabled": "Never", 
+        "id": "6e038c18-902f-4a41-b2a2-5f46fa62fc1b", 
+        "user": "Nobody"
```
Comment (ed-) [inline apidocs/src/resources/samples/db-mgmt-get-root-details-response.json:10]: Every time I see this I wonder about someone creating a user named Nobody. I wonder if it's too late to shift that to null or empty?

**Comment by ed-** [comment]
+1

**Comment by cp16net** [comment]
+1

**Pull request merged**